data/mn/legislature/Bobby-Joe-Champion-c7436be4-472a-4d5e-82f5-f46359395d3a.yml

```diff
@@ -13,11 +13,8 @@ roles:
   district: '59'
 offices:
 - classification: capitol
-  address: 2401 Minnesota Senate Bldg.; St. Paul, MN 55155
+  address: 3401 Minnesota Senate Bldg.; St. Paul, MN 55155
   voice: 651-296-9246
-- classification: district
-  address: 1501 Hall Curve N.;Minneapolis, MN 55411
-  name: 'District Office #1'
 links:
 - url: https://www.senate.mn/members/member_bio.html?mem_id=1212
```

Comment:
@bfossen-ce this person is the Senate President. I know we sometimes reflect legislator's leadership positions so I just wanted to note that in case we needed to add it.

Reply:
I think for MN we dont track this